atom/common/keyboard_util.cc

```diff
@@ -76,6 +76,7 @@ ui::KeyboardCode KeyboardCodeFromCharCode(base::char16 c, bool* shifted) {
     case '|': *shifted = true; case '\\': return ui::VKEY_OEM_5;
     case '}': *shifted = true; case ']': return ui::VKEY_OEM_6;
     case '"': *shifted = true; case '\'': return ui::VKEY_OEM_7;
+    case '±': *shifted = true; case '§': return ui::VKEY_OEM_8;
```

Comment:
This is failing to build with a compiler error:

```
../../atom/common/keyboard_util.cc:79:10: error: character too large for enclosing character literal type
    case '±': *shifted = true; case '§': return ui::VKEY_OEM_8;
         ^
../../atom/common/keyboard_util.cc:79:38: error: character too large for enclosing character literal type
    case '±': *shifted = true; case '§': return ui::VKEY_OEM_8;
```

See https://travis-ci.org/electron/electron/jobs/185833672 for more details.